mmv1/templates/terraform/custom_flatten/alloydb_cluster_input_user_flatten.go.erb

```diff
@@ -0,0 +1,22 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *Config) interface{} {
```

Comment:
this seems like a pretty standard flattener - what's different from the default generated code?

Reply:
these are not returned from api so there is a diff and these are force new /:

Reply:
oh right, because it's reading from state instead of the object. Gotcha. Thanks!